README.md

```
@@ -2,313 +2,1411 @@
-* [Mitul Savani](http://mitulsavani.com)
-* [Mohamed Abdel Nasser](https://www.mohdabdelnasser.live/)
-* [Mohammad Darmousa](https://mdarmousa.github.io/)
-* [Mohammed Boudad](https://matrixersp.design/)
-* [Moritz Kornher](https://moritzkornher.de/)
-* [Mostafa Sameti](https://sameti.dev/)
-* [Mouad ZIANI](https://mouadziani.github.io/)
-* [Muhammad](http://muhammadraza.me)
-* [Muhammad Muhaddis](https://muhaddis.info)
-* [Muntadhar Haydar](https://muntadhar.net)
+
+- [Maciej Pulikowski](https://pulik.dev)[Software Engineer & Security Researcher | 8 X Google Hall Of Fame]
```

markdownlint-cli2 (0.18.1)

Reversed link syntax
(https://pulik.dev)